training/prep_test_env.py

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
I think this file can be removed in favor of using the newer version? - https://github.ibm.com/CODAIT/max-model-training/blob/master/starter-kit/utils/prepare_test_environment.py